### ISSUE TITLE
'Passive' event handler

### ISSUE THREAD

**Issue by godban** [wilddeer/stickyfill#50]
Sometimes I receive next warning from Chrome:

> Handling of 'wheel' input event was delayed for 261 ms due to main thread being busy. Consider marking event handler as 'passive' to make the page more responive.

Does it make sense to add `{passive: true}` to event listeners?

**Comment by danimalweb** [comment]
I get the same warning.

**Comment by diegocanal** [comment]
Same here. I wonder what @wilddeer has to say about this.

> "Handling of 'wheel' input event was delayed for 243 ms due to main thread being busy. Consider marking event handler as 'passive' to make the page more responsive."

**Comment by wilddeer** [comment]
I used wheel event listener as a hack to avoid flickering during sticky state changes in Chrome. It should’t introduce any responsiveness problems, despite the warning. Adding `{passive: true}` would defeat the purpose of this event handler. 

I removed the wheel event listener completely from the new version of Stickyfill. Chrome now supports sticky positioning natively, and for older versions of Chrome (tiny percent of all Chrome users) default polyfill behavior without “wheel enhancement” is good enough to not bother with keeping the wheel hack.
